lib/nerves_hub_link/configurator.ex

```diff
@@ -39,6 +40,8 @@ defmodule NervesHubLink.Configurator do
             nerves_key: any(),
             params: map(),
             remote_iex: boolean(),
+            request_fwup_public_keys: boolean(),
+            shared_secret: [{String.t(), String.t()}],
```

Comment:
```suggestion
            shared_secret: [product_key: String.t(), product_secret: String.t()],
```
These are currently expected as atoms, correct?